README.md

```diff
@@ -1,2 +1,168 @@
+
+> The Tech stack that were used in this project are HTML & CSS.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML & CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- The website is responsive
+- Utilized Flexbox
+- Github workflow linter(HTML+CSS)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Anyone can use this project to learn how to setup a responsice website and linter for HTML-CSS files.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<ul>
+    <li><a href="https://git-scm.com/downloads">Git</a></li>
+    <li><a href="https://code.visualstudio.com/">VS Code</a></li>
+    <li><a href="https://nodejs.org/en/download/package-manager">Node</a></li>
+    <li>Chrome or any browser</li>
+    <li>Knowledge about Webdevelopment & Flexbox</li>
+</ul>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+cd Microverse_Project
+git clone https://https://github.com/Teklu2022/portfolio.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Mobile_Version_Project
+  npm i
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  use live server extension on VSCode and a Browser
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+npx hint .
+npx stylelint "**/*.{css,scss}"
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Teklu2022)
+- Twitter: [@twitterhandle](https://twitter.com/AddiseTeklu)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tekab-addise-897325227/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Responsive for Desktop
+- Contact forms
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project checkout my https://github.com/Teklu2022 for more projects.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and my coding partners for being my navigators in this awesome project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- Hey, please, kindly update the license link as it's currently broken to make your readme more professional.